cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-logging.adoc

```diff
@@ -0,0 +1,235 @@
+  name: cluster-logging
+  namespace: openshift-logging 
+spec:
+  targetNamespaces:
+  - openshift-logging
+----
+
+. Run the following script to configure your ROSA cluster to forward logs to CloudWatch:
++
+[NOTE]
+====
+Configuring ROSA to send logs to CloudWatch goes beyond the scope of this tutorial. Integrating with AWS and enabling CloudWatch logging are important aspects of ROSA, so a script has been included to simplify the configuration process. The script will automatically set up AWS CloudWatch. You can examine the script to understand the steps involved.
+====
++
+[source,terminal]
+----
+$ curl https://raw.githubusercontent.com/openshift-cs/rosaworkshop/master/rosa-workshop/ostoy/resources/configure-cloudwatch.sh | bash
+----
++
+.Example output
++
+[source,terminal]
+----
+deployment "cluster-logging-operator" successfully rolled out
+secret/cloudwatch-credentials created
+clusterlogforwarder.logging.openshift.io/instance created
+clusterlogging.logging.openshift.io/instance created
+----
+. After a few minutes, you should begin to see log groups inside of AWS CloudWatch. Run the following command to see the log groups:
++
+[source,terminal]
+----
+$ aws logs describe-log-groups --log-group-name-prefix rosa-${GUID}
+----
++
+.Example output
++
+[source,terminal]
+----
+{
+	"logGroups": [
+		{
+			"logGroupName": "rosa-fxxj9.audit",
+			"creationTime": 1682098364311,
+			"metricFilterCount": 0,
+			"arn": "arn:aws:logs:us-east-2:511846242393:log-group:rosa-fxxj9.audit:*",
+			"storedBytes": 0
+		},
+		{
+			"logGroupName": "rosa-fxxj9.infrastructure",
+			"creationTime": 1682098364399,
+			"metricFilterCount": 0,
+			"arn": "arn:aws:logs:us-east-2:511846242393:log-group:rosa-fxxj9.infrastructure:*",
+			"storedBytes": 0
+		}
+	]
+}
+----
+
+== Outputting the data to the streams and logs
+. Output a message to `stdout`.
+.. In the OSToy console, click *Home* and then click the message box for *Log Message (stdout)*.
+.. Write a message to output to the `stdout` stream, for example "All is well!".
+.. Click *Send Message*.
++
+image:cloud-experts-deploying-application-logging-ostoy-stdout.png[]
++
+. Output a message to `stderr`.
+.. Click the message box for *Log Message (stderr)*.
+.. Write a message to output to the `stderr` stream, for example "Oh no! Error!".
+.. Click *Send Message*.
++
+image:cloud-experts-deploying-application-logging-ostoy-stderr.png[]
+
+
+== Viewing the application logs by using the `oc` command
+. Enter the following command in the command line interface (CLI) to retrieve the name of your frontend pod:
++
+[source,terminal]
+----
+$ oc get pods -o name
+----
++
+.Example output
++
+[source,terminal]
+----
+pod/ostoy-frontend-679cb85695-5cn7x <1>
+pod/ostoy-microservice-86b4c6f559-p594d
+----
+<1> The pod name is `ostoy-frontend-679cb85695-5cn7x`.
++  
+. Run the following command to see both the `stdout` and `stderr` messages:
++
+[source,terminal]
+----
+$ oc logs ostoy-frontend-679cb85695-5cn7x
```

Comment:
Does it make more sense to have `oc logs <pod-name>` instead of the example one in the code to copy?